source/_integrations/opower.markdown

```diff
@@ -60,7 +60,7 @@ For gas:
 
 Because utilities only release usage/cost data with a 48-hour delay, the integration inserts data into statistic objects. At the initial setup, the integration pulls historical monthly usage/cost since the account activation, daily usage/cost for the past 3 years, and for electricity hourly usage/cost for the past 2 months. After the initial setup, the integration keeps pulling hourly data for electricity and daily data for gas for the past 30 days to allow for any corrections in the data from the utilities.
 
-In the configuration of the energy dashboard (**{% my energy title="Settings > Dashboards > Energy" %}**):
+In the configuration of the energy dashboard (**{% my config/energy title="Settings > Dashboards > Energy" %}**):
```

Comment:
`config/energy` isn't valid, as they cannot contain `/`.

According to:
https://my.home-assistant.io/create-link/?redirect=energy

The existing content is correct.

Reply:
The problem is it redirects to the energy dashboard and not to its configuration where you need to follow the steps below. Maybe remove the my link like I originally had in https://github.com/home-assistant/home-assistant.io/pull/26793 ?

Reply:
Aah I see:

`config_energy` is the right might key in that case.